src/modules/rc_update/rc_update.cpp

```diff
@@ -123,8 +123,10 @@ bool RCUpdate::init()
 	return true;
 }
 
-void RCUpdate::parameters_updated()
```

Comment:
Wow so until now when parameters were updated, this function was actually never called? (Since it's not overriding `updateParams`)

Reply:
It was manually called before otherwise stuff would not have worked properly. I removed the separate call since it's now going through the `updateParams()` mechanism. See: https://github.com/PX4/PX4-Autopilot/commit/76289d92f24d04bed89a74b9ca1208b586f10bc9#diff-bc65e8936281291b7e9801b185ee1740a6f2d4207e3b539c059469daa6cdfcc9L391

Reply:
I'm often confused about this `updateParams` (I guess the correct way to do it is by overriding this, as handled already in ModuleBase) or `parameters_updated`, or `update_parameters` scattered around the codebase 😆